docs/protocol-spec.md

```diff
@@ -89,7 +138,6 @@ A bindle that is marked `yanked = true` MUST be treated according to the followi
 - It MUST NOT be accepted by a `POST` operation
 - The `DELETE` operation is a no-up on a yanked Bindle
 - A `GET` request should only be fulfilled if the `yanked=true` query parameter is set. In any other case, it should mark it as "access denied"
-    - If `yanked=true` in the query string, the server SHOULD serve the bindle unaltered, including the `invoice.toml`'s `yanked = true` attribute.
```

Comment:
Any specific reason for removing this functionality?

Reply:
No functionality change here. This specific line is just redundant in this proposal; the invoice is _always_ immutable.